source/includes/secret_configs/_00-describe.md.erb

```diff
@@ -0,0 +1,18 @@
+<%=
+describe_object 'The secret config object', since: '19.4.0' do
+  id              'String', 'The identifier of the secret config'
+  pluginId        'String', 'The identifier of the plugin to which current secret config belongs'
+  configuration   'Array',  'The list of [configuration properties](#the-configuration-property-object) that represent the configuration of this secret config.'
+  rules           'Array',  'The list of [rules](#the-rule-object) that define where this secret config can be utilized.'
```

Comment:
How about?

The list of [rules](#the-rule-object), which allows restricting the usage of the SecretConfig. Referring to the SecretConfig from other parts of configuration is denied by default, an explicit rule should be added to allow a specific resource to refer the SecretConfig.